src/Sylius/Component/Core/Model/Image.php

```diff
@@ -35,6 +36,20 @@ class Image implements ImageInterface
     protected $path;
 
     /**
+     * Image title
```

Comment:
Thats strange.  I don't see this in the code:
https://github.com/pfwd/Sylius/blob/46f41758f5f2ccd02308fa1887c3b3179fa7137a/src/Sylius/Component/Core/Model/Image.php